server/controllers/comments.js

```diff
@@ -0,0 +1,162 @@
+import models from '@models';
+import validateComment from '@validations/comment';
+import comments from '@helpers/comments';
+import { validationResponse } from '@helpers/validationResponse';
+
+const { Comment } = models;
+
+/**
+ * @exports CommentController
+ * @class CommentController
+ * @description Handles the comment section
+ */
+class CommentController {
+  /**
+   * Create new comment
+   * @static
+   * @param {object} req - The Request Object
+   * @param {object} res - The Response Object
+   * @param {object} next - The Next Middleware
+   * @return {json} - Returns json Object
+   * @memberof CommentController
+   * @static
+   */
+  static async create(req, res, next) {
+    try {
+      const data = await validateComment(req.body);
+      const body = data.comment;
+      const { slug } = req.params;
+      const userId = req.decoded.id;
+      const { article } = req;
+
+      const comment = await article.createComment({ userId, slug, body });
+
+      const payload = await comments(article.id);
+
+      if (comment) {
+        return res.status(201).json({
+          status: 201,
+          message: 'Comment added successfully.',
+          payload
+        });
+      }
+    } catch (error) {
+      if (error.isJoi && error.name === 'ValidationError') {
+        return res.status(400).json({
+          status: 400,
+          errors: validationResponse(error)
+        });
+      }
+
+      next(error);
+    }
+  }
+
+  /**
+   * Get all comments
+   * @static
+   * @param {object} req - The Request Object
+   * @param {object} res - The Response Object
+   * @param {object} next - The Next Middleware
+   * @return {json} - Returns json Object
+   * @memberof CommentController
+   * @static
+   */
+  static async getComments(req, res, next) {
+    try {
+      const { article } = req;
+
+      const payload = await comments(article.id);
+
+      if (payload) {
```

Comment:
You probably should return the query ignoring the if statement

Reply:
True that. I have implemented your feedback.